.travis.yml

```diff
@@ -9,7 +9,6 @@ go:
 env:
   matrix:
     - GOTFLAGS="-race"
-    - GOTFLAGS="-race -tags=openssl"
```

Comment:
"fails with no output produced" is a reason to track down the bug and fix it. Let's not just remove a test.

Reply:
I have added a `travis_wait` and increased the test timeout, and things seem to be smooth now.
I can restore.

Reply:
I suspect the reason is that travis takes a very long time to build the tests for running, which is exacerbated with openssl.

Reply:
restored.